samples/apps/copilot-chat-app/webapi/DeploymentTemplates/DeploySK-Existing-AI.ps1

```diff
@@ -0,0 +1,74 @@
+# Script to create a Semantic Kernel service deployment using an
+# existing OpenAI or Azure OpenAI instance.
+
+param(
+    [Parameter(Mandatory)]
+    [string]
+    $DeploymentName,
+
+    [Parameter(Mandatory)]
+    [string]
+    $ApiKey,
+
+    [string]
+    $Endpoint = "",
+
+    [string]
+    $AIService = "AzureOpenAI",
```

Comment:
(nit) Do we need a validate set to avoid typo and catch unsupported service during setup?
e.g. `[ValidateSet(“AzureOpenAI”,”OpenAI”)]`

Reply:
Added